modules/realvuAnalyticsAdapter.js

```diff
@@ -0,0 +1,975 @@
+// RealVu Analytics Adapter
+import adapter from 'src/AnalyticsAdapter';
+import adaptermanager from 'src/adaptermanager';
+import CONSTANTS from 'src/constants.json';
+
+const utils = require('src/utils.js');
+
+var realvuAnalyticsAdapter = adapter({
+  global: 'realvuAnalytics',
+  handler: 'on',
+  analyticsType: 'bundle'
+});
+/*
+  Copyright(C) 2001-2018, RealVu Inc., All rights reserved. Use of RealVu’s patented, patent pending,
```

Comment:
code submitted to the project is governed by the prebid license. You cannot add a separate license here.

Reply:
@mkendall07 ,  RealVu has looked into this with our legal counsel and believe that it is acceptable, appropriate and necessary to include this copyright protection notice in the adapter.  Our objective is to protect our intellectual property.  We are not contributing our proprietary code to the upstream project.  This notice excludes it from the Apache license.  If you believe it is still unacceptable, could you please let us know who we can contact at Prebid.org to further this discussion?
Thx,
-mark

Reply:
@markhe 
The recommended approach here would be to put your proprietary code into an external js file and load it from the analytics adapter. You may include the license on that file. Thanks